hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -97,16 +97,23 @@ class HassioIngressView extends LitElement {
             title: requestedAddon,
           });
           await nextRender();
-          history.back();
+          navigate("/hassio/store", { replace: true });
           return;
         }
-        if (!addonInfo.ingress) {
+        if (!addonInfo.version) {
+          await showAlertDialog(this, {
+            text: this.supervisor.localize("my.error_addon_not_installed"),
+            title: addonInfo.name,
+          });
+          await nextRender();
+          navigate(`/hassio/addon/${addonInfo.slug}/info`, { replace: true });
+        } else if (!addonInfo.ingress) {
           await showAlertDialog(this, {
             text: this.supervisor.localize("my.error_addon_no_ingress"),
             title: addonInfo.name,
           });
           await nextRender();
-          history.back();
+          navigate(`/hassio/addon/${addonInfo.slug}/info`, { replace: true });
```

Comment:
If it has a web ui, should we go there?

Reply:
I wonder now if we should build such logic into the ingress vote actually so it works when people have a panel too

Reply:
Even if it has a webUI, that experience is not great, it only works locally

Reply:
In my setup it never works, as my HA is behind a reverse proxy 🤷‍♂️